Source/WebCore/style/values/color/StyleColor.h

```diff
@@ -125,7 +125,7 @@ struct Color {
     WEBCORE_EXPORT Color(const Color&);
     Color& operator=(const Color&);
 
-    Color(Color&&);
+    WEBCORE_EXPORT Color(Color&&);
```

Comment:
I always wonder if exporting functions makes them non-inlineable, and hurts perf.

Reply:
I don’t believe it does, but it sure is annoying.